src/Utils.php

```diff
@@ -216,7 +216,7 @@ private static function scalarToString($value): string
     }
 
     /**
-     * @param array<mixed> $value
+     * @param array<mixed, mixed> $value
```

Comment:
I would favor `array-key` over `mixed` for the key type. Since it's already what `array<T>` implies, IMO it's OK to keep the key type implicit but I'm fine with explicit type too.

Reply:
```suggestion
     * @param array<array-key, mixed> $value
```

I do not want to keep implicit type for key anywhere for the project, as then I'm guessing if it's list or not. with explicit `list<type>` usage, i know exactly it's list, and with explicit usage of `array<key, val>`, i know it's not. with `array<val>`/`val[]`, I'm guessing.
for that, i wanted to add key here.